### PR TITLE
Docker publish develop branch to dev environment.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: src/bin/travis/afterSuccess.sh [hunk 1]
```diff
@@ -9,10 +9,11 @@ echo "TRAVIS_PULL_REQUEST='$TRAVIS_PULL_REQUEST'"
 if [ "$BUILD_TYPE" == "sbt" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
 
     if [ "$TRAVIS_BRANCH" == "develop" ]; then
+        # Publish images for both the "cromwell develop branch" and the "cromwell dev environment".
         docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
         sbt \
           'set test in Test := {}' \
-          'set imageNames in docker := Seq(ImageName("broadinstitute/cromwell:'${TRAVIS_BRANCH}'"))' \
+          'set imageNames in docker := Seq("develop", "dev").map(tag => ImageName(s"broadinstitute/cromwell:$tag"))' \
```
Comment (Horneth) [inline src/bin/travis/afterSuccess.sh]: Could it be `dev-env` or something alike ? Just to avoid having `dev` be confused with a shortened `develop`

**Comment by Horneth** [comment]
👍

File: src/bin/travis/afterSuccess.sh (hunk 1 above)
Comment (kshakir) [inline src/bin/travis/afterSuccess.sh]: No, or at least not yet. And yes it is confusing, but after three months one shrugs and gets used to environments-vs.-branches. The names of the environments come from our devops friends. We'd have to change the environment names in several other places.

**Comment by danbills** [comment]
👍

**Pull request merged**